src/components/App/AppComponent.js

```diff
@@ -20,6 +21,9 @@ export class AppComponent extends PureComponent {
       case PAGES_ENUM.NOTES:
         return <NotesPageContainer />;
 
+      case PAGES_ENUM.NOTE_DETAILS:
+        return <NoteDetailsPage />;
```

Comment:
This PR says "Enforce eslint" but it adds `NoteDetailsPage`. PR "Delete redundant code" also adds `NoteDetails`Page https://github.com/DarthVanger/NeuralNotes/pull/165/files#diff-bbecf9f29e0e6600adcce7696083e843997dbd6fa01c10f6ebc2ea95e1cea7bfR25

I'm confused, what should I review? Why are you introducing the same changes in multiple PRs?

Reply:
Each PR based on the previous. When you will add first PR you will not have duplicates